travisci/deltaspike-v1.x.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Current strategy is, for each supported minor version of DeltaSpike, to test one early point release plus
+# the most recent point release, plus some point releases which caused problems.
+DELTASPIKE_VERSIONS=(
```

Comment:
"For each supported minor version of DeltaSpike, we test the most recent point release." At least, that's what seems to appear below.

Reply:
Updated in https://github.com/cdi-unit/cdi-unit/commit/15f583a64588f60c9b2c889495a58e580cf7cfee